pgpool/configs/pgpool.conf

```diff
@@ -385,7 +385,7 @@ failback_command = ''
                                    #   %R = new master database cluster path
                                    #   %% = '%' character
 
-fail_over_on_backend_error = off
+fail_over_on_backend_error = on
```

Comment:
We already do dynamic pgpool configuration, maybe we can define this option via ENV?
Check `pgpool/bin/pgpoo_setup.sh`
@paunin what do you think?

Reply:
that option should not be in 'on' if you use 
http://www.pgpool.net/docs/latest/en/html/runtime-config-health-check.html

Reply:
but you still can use CONFIGS, up to you to configure your pgpool instance
https://github.com/paunin/PostDock/blob/master/docker-compose.yml#L171